src/overlays/actors/ovl_Shot_Sun/z_shot_sun.c

```diff
@@ -14,7 +33,17 @@ void ShotSun_Init(Actor* thisx, GlobalContext* globalCtx);
 void ShotSun_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void ShotSun_Update(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+void ShotSun_FairyCountdown(ShotSun* this, GlobalContext* globalCtx);
+void ShotSun_StartFairyCountdown(ShotSun* this, GlobalContext* globalCtx);
+void func_80BADF0C(ShotSun* this, GlobalContext* globalCtx);
+void ShotSun_HyliaSunUpdate(ShotSun* this, GlobalContext* globalCtx);
+
+s32 func_8005B198();
+void func_80062718(ColliderCylinder* collider, Vec3s* pos);
+void func_80078884(u16 sfxId);
```

Comment:
for functions outside of the file (that are accessed globally) we have functions.h. youll want to put these there.
the third one here func_80078884 is already in functions.h

Reply:
Where exactly in there should I put them?

Reply:
the file is in order by address. lots of functions will already have an entry there just commented out. if they dont then add it in

Reply:
done